itou/templates/dashboard/includes/stats_kpi_labor_inspector_iae.html

```diff
@@ -0,0 +1,78 @@
+{% load humanize %}
+
+<section class="s-section mb-3 mb-md-5">
+    <div class="s-section__container">
+        <div class="row mb-1 mb-md-3">
+            <div class="col-12">
+                <h2 class="mb-0">Chiffres clés</h2>
+                <p class="fs-sm mb-3">
+                    Dernière mise à jour des données du flux IAE : {{ stats_kpi.flux_iae_data_updated_at|date:"d F Y"|default:"Jamais" }}
+                </p>
+            </div>
+            <div class="col-12">
+                <div class="d-flex flex-column flex-lg-row gap-3 gap-lg-4 p-3 p-lg-4 bg-light rounded-3">
+                    {# job_seeker_still_seeking_after_30_days #}
+                    <div class="d-flex flex-column w-100">
+                        <div class="flex-grow-1">
+                            <p class="mb-0 mb-lg-3">
+                                <i class="ri-bar-chart-line ri-2x text-pilotage" aria-hidden="true"></i>
+                                <span class="h1">{{ stats_kpi.job_seeker_still_seeking_after_30_days|default:"N/A"|intcomma }}</span>
+                            </p>
+                            <p>C’est le nombre de candidats dans la file active IAE de votre territoire à l’instant.</p>
+                        </div>
+                        <div>
+                            <a href="https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/bilan-candidatures-iae/"
+                               target="_blank"
+                               rel="noopener"
+                               aria-title="En savoir plus sur ces données"
```

Comment:
Je pense il devrait être aria-label ([le WAI ne recommande pas utiliser `title` sans `label`](https://www.w3.org/WAI/tutorials/forms/labels/#using-the-title-attribute))

```suggestion
                               aria-label="En savoir plus sur ces données (ouverture dans un nouvel onglet)"
                               aria-title="En savoir plus sur ces données"
```